packages/archives.sls

```diff
@@ -2,7 +2,7 @@
 # vim: ft=sls
 {% from "packages/map.jinja" import packages with context %}
 
-{% set req_packages = packages.pkgs.required.pkgs + ['curl', 'tar', 'bzip2', 'gzip',] %}
+{% set req_packages = packages.pkgs.required.pkgs + ['curl', 'bzip2', 'gzip',] %}
```

Comment:
How about moving these to the `defaults.yaml`, `osfamilymap.yaml`, etc files? they'll be merged my `map.jinja`, right?

Reply:
Sure thats best idea. thx